app/models/concerns/client_image_consumer.rb

```diff
@@ -50,6 +50,9 @@ def local_client_image_data
       headshot = uploaded_local_image
```

Comment:
I was curious about the requirement that an ETO api be available, in this file on line 18?

Reply:
Ah, that's because that's the only API we have historically integrated with.  Looks like we check it again on line 27.  We probably need to pull out the call on line 18.  Thanks!  I'd prefer to wrap the ETO specific logic into a method or ruby class.  But we can do that later.

Reply:
Oh, I see, sorry, to waffle so much, that was limiting it so no images show up unless they "have" the ETO API...  There are definitely some installations where it doesn't make sense to have a fake images.  Let's try this for now, but we may need some other configuration to check if we should just hide images, even in staging.

Reply:
Ah, ok. Should I revert that removal?